README.md

```diff
@@ -1,6 +1,13 @@
 # OpenShift Application Services Connection Operator (RHOAS-Operator)
 
-### Documentation
+OpenShift Application Services Connection Operator (RHOAS-Operator) manages the connections between RHOAS Service Accounts and RHOAS Cloud Services. Users provide RHOAS-Operator with a [cloud.redhat.com](https://cloud.redhat.com) API Token via a secret and the operator uses that token to access the Managed Services APIs on behalf of the user to process their KafkaConnection, CloudServicesRequest, and CloudServiceAccountRequest custom resources. These resources are monitored by the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) which provides connectivity between your application and RHOAS services.
+
+## Related Projects
+
+ * [app services cli](https://github.com/redhat-developer/app-services-cli)
+ * [app services console plugin](https://github.com/redhat-developer/app-services-openshift-console-plugin)
```

Comment:
```suggestion
 * [OpenShift UI extension for RHOAS operator](https://github.com/redhat-developer/app-services-openshift-console-plugin)
```

Reply:
This one is tricky. My take is that we can actually remove this linked and say - RHOAS operator enables extra capabilities in the OpenShift UI console